README.md

```diff
@@ -173,12 +173,11 @@ You can create a collection based on the Plex search feature using the `plex_sea
 - `actor` (Gets every movie with the specified actor) (Movie libraries only)
 - `tmdb_actor` (Gets every movie with the specified actor as well as the added TMDb [metadata](#tmdb-people-list-type)) (Movie libraries only)
 - `country` (Gets every movie with the specified country) (Movie libraries only)
-- `decade` (Gets every movie from the specified year + the 9 that follow i.e. 1990 will get you 1990-1999) (Movie libraries only)
 - `director` (Gets every movie with the specified director) (Movie libraries only)
 - `tmdb_director` (Gets every movie with the specified director as well as the added TMDb [metadata](#tmdb-people-list-type)) (Movie libraries only)
 - `genre` (Gets every movie/show with the specified genre)
 - `studio` (Gets every movie/show with the specified studio)
-- `year` (Gets every movie/show with the specified year)
+- `year` (Gets every movie/show with the specified year) (Put a `-` between two years for a range i.e. `year: 1990-1999` or end with `NOW` to go till current i.e. `year: 2000-NOW`)
```

Comment:
Can we keep undocumented support for `decade`?

Reply:
well i took out decade in favor of the year range i added just do `year: 1990-1999` i can add it back in if needed but i figured the range would be less confusing and not bog down the user with another option plus the range isn't limited to 10 years either

Reply:
Did you want me to add decade back or is year range ok?

Reply:
I prefer to keep it

Reply:
ok ill add it back silently